### ISSUE TITLE
Todour 2.19: Editing tasks deletes creation date

### ISSUE THREAD

**Issue by Moonbase59** [SverrirValgeirsson/Todour#25]
I have set
[x] Add dates to lines
[ ] Show dates

Dates aren’t shown in the GUI (correct).
When creating a new task, a date gets added in the file (correct).
But when _editing_ an existing entry, the creation date is REMOVED in the file!

**Comment by sverrirv-sib** [comment]
Hi. That was not exactly as designed..  Thanks. 
What do you think of this as a solution (I am being convenient, but also it makes it possible to edit lines better):

A line is shown like this from start:
![image](https://user-images.githubusercontent.com/75308383/109393262-f1592e00-7920-11eb-98e7-2c6eef629f08.png)

Then when editing:

![image](https://user-images.githubusercontent.com/75308383/109393297-151c7400-7921-11eb-8dbd-a50109bd1f4f.png)

And then when done:

![image](https://user-images.githubusercontent.com/75308383/109393308-25ccea00-7921-11eb-9289-4c8c0c8d1530.png)

So basically showing all the dates in the line when editing (allowing the user to also change that in case that is of interest)

**Comment by SverrirValgeirsson** [comment]
Oops. Was logged in on work account. But it was me writing above :)

**Comment by Moonbase59** [comment]
Yep, I guessed so ;-)

You have a point there. It might actually be good to show the *real* line (as in the file) when editing. Then, for displaying, hiding what’s currently unwanted.

**Comment by SverrirValgeirsson** [comment]
I will let that go in then. Though found a similar issue with that setting that if an item is checked and then unchecked again the date also dissapears. I will fix that as well before comitting this. 

I use this software every day and made it for myself. Just fun that there are others that have any use of it :)

**Comment by SverrirValgeirsson** [comment]
Fixed in 99833912cc3a3d5c84f78d9a1788548f6ee2587f

**Comment by Moonbase59** [comment]
Thanks for taking care and listening to users!

I actually sync my _todo.txt_ between Android smartphone, Linux laptop and desktop using Syncthing, and use _Todour_ on both laptop and desktop.
